src/main/kotlin/com/atlassian/performance/tools/report/api/result/StatsMeter.kt

```diff
@@ -112,4 +181,53 @@ class StatsMeter {
         )
         return data.durationMapping(measurement)
     }
-}
\ No newline at end of file
+
+    private fun extractOutlierBy(actionToOutlier: Map<ActionType<*>, OutlierTrimming>,
+                                 metricLabel: String)
```

Comment:
Perhaps we could work on `type: ActionType<*>` rather than `metricLabel: String` and simplify all of this to `actionToOutlier[type]`.

Reply:
Good point, but I refactored the code so this method doesn't even exist anymore.